deployment/deployment.dev.json

```diff
@@ -0,0 +1,3634 @@
+{
```

Comment:
is this file supposed to be apart of this change?

Reply:
Yeah - I am not sure how that ended up there. For some reason I added the deployment.dev.json back in, but we do not want it.